docs/dietpi_tools.md

```diff
@@ -779,30 +805,98 @@ dietpi-backup
+    - "On": In this case all files previously existing in the target directory which do not exist in the source directory will be deleted to achieve an exact copy of your source directory
+
+=== "Automatic daily sync"
+
+    `Dietpi-Sync` gives the option of an automatic daily sync operation (controlled via the Linux `cron` mechanism) by enabling the "Daily Sync" option.
+
+    **Daily sync execution time**
+
+    The automatic daily sync is controlled via the Linux `cron` mechanism. Setting a different starting time can be an option, e.g. if you have several sync clients syncing up to the same storage (data server): Shifting the synchronization starting time of these systems may reduce temporary overload of the data server by avoiding concurrent access to the storage.
+
+    The starting time is basically defined via the file `/etc/crontab` (which calls the sync function via the `/etc/cron.daily/dietpi` script). It can be changed via the entry `cron.daily` within [`dietpi-cron`](#dietpi-cron). It is executed by running the following command
+
+    ```sh
+    dietpi-cron
+    ```
+
+    Please keep in mind that all other daily `cron` based procedures are also started at this changed time.
+
+=== "Sync file selection (Filter)"
+
+    The definition which files are used for the synchronization procedure is defined via the file 
+    
+    ```
+    /boot/dietpi/.dietpi-sync_inc_exc
+    ``` 
+    
+    This file can be edited to set further include/exclude definitions for the synchronization. The filter definition syntax is described within the file itself.
+
+    The file structure definition is identical to the DietPi backup/restore file `/boot/dietpi/.dietpi-backup_inc_exc` where it is explained more in detail.
+
+=== "Logging"
+
+    Logging information about the synchronization procedure is given within the file  
+    
+    ```
+    .dietpi-sync.log
+    ```
+
+    which is written to the sync target directory. It gives a list of every processed file.
+
+    The execution status of a previous synchronization process is given in the "Last sync status" entry at the top of the `dietpi-sync` dialog.
+
+=== "Settings file"
+
+    Generally, the settings of the DietPi-Sync are changed via the `dietpi-sync` command menu entries.
+
+    The system stores these settings in the file `/boot/dietpi/.dietpi-sync_settings`, which is generated from `dietpi-sync` automatically. Therefore, the file do not need to be changed manually by the user.  
+    An example settings file is:
+
+    ```
+    FP_SOURCE='/mnt/source'
+    FP_TARGET='/mnt/target'
+    SYNC_DELETE_MODE=0
+    SYNC_CRONDAILY=0
+    ```
+
+=== "Scripted run"
+
+    DietPi-Sync can be run from the command line or from scripts without user interaction by calling it via 
+    
+    ```sh
+    dietpi-sync 1
+    ```
+
+=== "Dry run"
+
+    Each sync includes a leading dry run, after which you can check the expected result before deciding if you want to continue with the actual sync:
```

Comment:
Unrelated to this PR, but shall we actually keep the forced dry run + space check here? We removed it in `dietpi-backup` since it can take quite long and the space check may show imprecise free space results on network drives, and a failing sync because of insufficient free space is usually not critical. Instead we added an option to enable/disable the dry run, which we could do here as well.

Reply:
I also thought about it to keep it in sync with dietpi-backup. But as long as it is not implemented, the docu shall describe the actual status.
If this will be changed in dietpi-sync, the docu will also be changed in parallel. :-)

Reply:
I'll implement the change/alignment with next release.

Reply:
Then pls don't forget an issue for the Docs. :-)